src/tests/test_integration.py

```diff
@@ -21,7 +21,7 @@
 __all__ = ()
 
 
-class TestEnv(object):
+class FakeTestEnv(object):
```

Comment:
Oh, this is because `pytest` tries to collect this class? Then it's fine to change the name. However, I don't like "fake" because is creates a real file system environment. How about `InvocationEnv` / `StagingEnv` / `DeployEnv`, etc.?